neoexchange/astrometrics/tests/test_sources_subs.py

```diff
@@ -415,7 +416,36 @@ def test_page_down(self):
         self.assertEqual(expected_targets, targets)
 
 
-class TestFetchGoldstoneTargets(TestCase):
+class TestFetchGoldstoneCSV(SimpleTestCase):
+
+    def setUp(self):
+        self.test_file = os.path.join('astrometrics', 'tests', 'test_goldstone_page.csv')
+
+    def test_basic(self):
+        expected_length = 7
+        expected_columns = ['number', 'name', 'start (UT)', 'end (UT)', 'OCC', 'Updated 2021 Dec 17']
+
+        table = fetch_goldstone_csv(self.test_file)
+
+        self.assertEqual(expected_length, len(table))
```

Comment:
Shouldn't the expected length be 9? I don't think do any cuts yet.

Reply:
also, this test fails for me.

Reply:
Fixed in 2d71c1a556cf0b2ac79a6defcce29b31aff71bc0